scripts/verify-OSCAL-contents.sh

```diff
@@ -0,0 +1,164 @@
+#!/bin/sh
+set -e
+
+# This script verify OSCAL contents using complyctl and OpenSCAP,
+# make sure OSCAL align with OpenSCAP in rules.
+# https://issues.redhat.com/browse/CPLYTM-254
+# usage: ./scripts/verify-OSCAL-contents.sh "$catalogs" "$profiles" "$component_definitions" "$product_name"
+
+# OSCAL Catalogs relative path string, multiple elements separate by space
+catalogs=$1
+# OSCAL Profiles relative path string, multiple elements separate by space
+profiles=$2
+# OSCAL Component Definitions relative path string, multiple elements separate by space
+component_definitions=$3
+# test product name
+product=$4
+
+TRESTLE_PREFIX="trestle://"
+RULE_ALIGNMENT_THRESHOLD_PERCENTAGE=60
+# array to store verified files, avoid testing one file multiple times
+verified_files=()
+
+# A function to check if an element exists in an array.
+# Returns 0 if found, 1 if not found.
+# Usage: contains_element "element_to_find" "${array[@]}"
+contains_element() {
+  local element_to_find="$1"
+  shift # Remove the first argument so that the rest are the array elements
+  local array_to_search=("$@")
+
+  for element in "${array_to_search[@]}"; do
+    if [[ "$element" == "$element_to_find" ]]; then
+      return 0 # Found the element
+    fi
+  done
+
+  return 1 # Element was not found
+}
+
+# setup OSCAL content for complyctl
+setup_complyctl_files() {
+  local catalog_path="$1"
+  local profile_path="$2"
+  local cd_path="$3"
+  # get framework id
+  framework_id=$(jq -r '.["component-definition"].components[0]."control-implementations"[0].props[] |
+   select(.name="Framework_Short_Name").value' "$cd_path")
+  # clean complyctl dir
+  rm -f /usr/share/complytime/bundles/*
```

Comment:
How about replacing the directory `/usr/share/complytime` with a constant, like "DIR" or "workspace"? It was used many times. WDYT?